web/ui/static/vendor/bootstrap3-typeahead/bootstrap3-typeahead.min.js

```diff
@@ -1,21 +1 @@
-/* =============================================================
- * bootstrap3-typeahead.js v3.1.0
```

Comment:
@jritchieBAE I notice that the typeahead's license header and info got removed in https://github.com/prometheus/prometheus/pull/5192. I think we probably want to keep (re-add) that, so I added another commit to add the latest license header from https://github.com/bassjobsen/Bootstrap-3-Typeahead/blob/master/bootstrap3-typeahead.js.